script/travis/mysql.sh

```diff
@@ -4,16 +4,16 @@ set -e
 # 1. Use READ-COMMITTED transaction isolation level
```

Comment:
Can you update this comment so it's no longer a list, removing the mention of the Barracuda file format that we no longer manually set?